Sources/App/Commands/TriggerBuilds.swift

```diff
@@ -262,7 +262,11 @@ func triggerBuildsUnchecked(on database: Database,
                             logger: Logger,
                             triggers: [BuildTriggerInfo]) -> EventLoopFuture<Void> {
     triggers.flatMap { trigger -> [EventLoopFuture<Void>] in
-        logger.info("Triggering \(trigger.pairs.count) builds for package name: \(trigger.packageName), ref: \(trigger.reference)")
+        if let packageName = trigger.packageName, let reference = trigger.reference {
+            logger.info("Triggering \(pluralizedCount: trigger.pairs.count, singular: "build") for package name: \(packageName), ref: \(reference)")
+        } else {
+            logger.info("Triggering \(pluralizedCount: trigger.pairs.count, singular: "build") for version ID: \(trigger.versionId)")
+        }
```

Comment:
This little rabbit hole opened up when fixing the "builds" label here and regretting that `pluralizedCount` was a top level function.